src/components/modalWindowContent/modalWindowContent.js

```diff
@@ -0,0 +1,281 @@
+  const theadTr = document.createElement('tr');
+  theadTr.classList.add('table_theadTr');
+
+  const columnNames = ['', 'You', 'Computer', 'Answer'];
+  columnNames.forEach((name) => {
+    const th = document.createElement('th');
+    th.innerText = `${name}`;
+    theadTr.appendChild(th);
+  });
+
+  thead.appendChild(theadTr);
+  table.appendChild(theadTr);
+
+  const tableBody = document.createElement('tbody');
+  tableBody.classList.add('table_body');
+
+  answersList.forEach((answer) => {
+    const markup = `
+        <tr> 
+          <td> <img src="${answer.img}" class="table_img"/> </td>
+          <td class="table-answer--${
+            answer.human.isCorrect ? 'correct' : 'notcorrect'
+          }"> ${answer.human.answer} </td>
+          <td class="table-answer--${
+            answer.autoPlayer.isCorrect ? 'correct' : 'notcorrect'
+          }"> ${answer.autoPlayer.answer} </td>
+          <td class="table-answer--default"> ${answer.correctAnswer} </td>
+        </tr>`;
+    tableBody.insertAdjacentHTML('beforeend', markup);
+  });
+
+  table.appendChild(tableBody);
+  return table;
+};
+
+const createNameInput = () => {
+  const nameInput = document.createElement('input');
+  nameInput.classList.add('form_name-input');
+  nameInput.setAttribute('id', 'name');
+  nameInput.setAttribute('placeholder', 'Type your name here');
+  return nameInput;
+};
+
+const createNameLabel = () => {
+  const nameLabel = document.createElement('label');
+  nameLabel.classList.add('form_name-label');
+  nameLabel.setAttribute('for', 'name');
+  nameLabel.innerText =
+    'Please fill your name in order to receive eternal glory in whole Galaxy!';
+  return nameLabel;
+};
+
+const createWarning = () => {
+  const warning = document.createElement('span');
+  warning.innerText = 'This field cannot be empty';
+  warning.classList.add('form_warning');
+  return warning;
+};
+
+const showWarning = () => {
+  const warning = document.querySelector('.form_warning');
+  warning.classList.add('form_warning--active');
+};
+
+const hideWarning = () => {
+  const warning = document.querySelector('.form_warning');
+  warning.classList.remove('form_warning--active');
+};
+
+const createSubmitButton = () => {
+  const submitBtn = document.createElement('button');
+  submitBtn.classList.add('form_submitBtn');
+  submitBtn.setAttribute('type', 'submit');
+  const submitTxt = 'may the force be with you';
+  submitBtn.innerText = submitTxt.toUpperCase();
+  return submitBtn;
+};
+
+const handleSubmit = (e) => {
+  e.preventDefault();
+  const input = document.querySelector('.form_name-input');
+  const playerName = input.value;
+  if (playerName === '') {
+    showWarning();
+  } else {
+    const score = countCorrectAnswers(answersList, 'human').length;
+    saveScore(playerName, score);
+    input.value = '';
+    setTimeout(() => {
+      hideModal();
+    }, 1000);
+  }
+};
+
+export const modalContent = (answersList, saveScore) => {
```

Comment:
`endGameModal`, `gameResultsModal` or something similar

Reply:
`saveScore` is not used here or am I blind?

Reply:
Don't worry, you have eyes like an eagle! `saveScore` is supposed to be passed as a callback / `Komponent przyjmuje na wejściu także funkcję, która zostanie wywołana po naciśnięciu przycisku` and `Komponent wyświetla przycisk, którego naciśnięcie powoduje wywołanie przekazanej funkcji. Funkcja jako argument otrzymuje wpisany nick gracza oraz jego punktację.`/. However I forgot to pass it to handle function. I am changing it to 
```submitBtn.addEventListener('click', (e) =>
    handleSaveUserResult(e, saveScore)
  );```